raiden/tests/scenarios/mfee3_imbalance_fee.yaml

```diff
@@ -0,0 +1,98 @@
+version: 2
+
+settings:
+  gas_price: "fast"
+  chain: any
+  services:
+    pfs:
+      url: https://pfs-goerli-with-fee.services-dev.raiden.network
+    udc:
+      enable: true
+      token:
+        deposit: true
+
+token:
+  address: "0x62083c80353Df771426D209eF578619EE68D5C7A"
+  balance_min: 1_000_000
+  balance_fund: 1_000_000
+
+nodes:
+  mode: managed
+  count: 3
+  raiden_version: local
+
+  default_options:
+    gas-price: fast
+    environment-type: development
+    routing-mode: pfs
+    pathfinding-max-paths: 5
+    pathfinding-max-fee: 100
+    flat-fee:
+      - "0x62083c80353Df771426D209eF578619EE68D5C7A"
+      - 0
+    proportional-fee:
+      - "0x62083c80353Df771426D209eF578619EE68D5C7A"
+      - 0
+    proportional-imbalance-fee:
+      - "0x62083c80353Df771426D209eF578619EE68D5C7A"
+      - 10_000
+
+## This is the MFEE3 scenario. It creates a network with topology 0 -> 1 -> 2 and checks
```

Comment:
Silly question. What's the difference between one and two `#`

Reply:
No difference. Just an old inconsistency that we did get rid off I think.

Reply:
Then perhaps remove the extra `#` just for consistency